backend/test/performanceTest.py

```diff
@@ -176,12 +176,9 @@ def generate_users():
 
 ######################### Run tests #########################
 
-generate_data = input("Generate Database Data (y/n): ")
-
-if generate_data == "y":
-  print("Generating data...")
-  generate_goals()
-  generate_users()
+# Goals and users already generated on test server
```

Comment:
Are these commented out because this development server will always have the goals in the db?

Should we remove the functions generate_goals and generate_users?

Reply:
Sure I'll remove that and just mention it in comments